coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/PlaybackToSecondary.scala

```diff
@@ -380,6 +380,10 @@ class PlaybackToSecondary[CT, CV](u: PlaybackToSecondary.SuperUniverse[CT, CV],
     }
 
     def updateSecondaryMap(newLastDataVersion: Long): Unit = {
+      // We want to end the current transaction here. We don't want to be holding share locks on data-tables like log
```

Comment:
nit, can you add commenting that all db activities up to the first commit should be read only?